BE/Startup.cs

```diff
@@ -1,67 +1,91 @@
-using System.Linq;
 using Microsoft.AspNetCore.Builder;
 using Microsoft.AspNetCore.Hosting;
-using Microsoft.AspNetCore.Http;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Hosting;
 using Microsoft.EntityFrameworkCore;
-using System.Text;
+using System;
+using Microsoft.Extensions.Configuration;
+using Microsoft.OpenApi.Models;
+using BE.Queries.Revisions.GetAllRevisions;
+using BE.Queries.Revisions.GetRevisionById;
+using BE.BL.Revisions.Create;
+using BE.BL.Revisions.Edit;
+using BE.Queries.Cars;
+using BE.BL.Cars.Create;
+using BE.BL.Cars.Edit;
+using BE.Queries.Revisions.GetAllRevisionsByClientId;
 
 namespace BE
 {
     public class MyContext : DbContext
-    {
-        protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
-        {
-            optionsBuilder.UseSqlServer(@"");
+    {    
+        public MyContext (DbContextOptions<MyContext> dbContextOptions) :base(dbContextOptions)
+        {           
         }
-
         protected override void OnModelCreating(ModelBuilder modelBuilder)
         {
             modelBuilder.Entity<MyClass>().HasNoKey();
+            modelBuilder.Entity<GetAllRevisionsResult>().HasNoKey();
+            modelBuilder.Entity<GetAllRevisionsByClientIdResult>().HasNoKey();
+            modelBuilder.Entity<GetRevisionByIdResult>().HasNoKey();
+            modelBuilder.Entity<RevisionCreateCommand>().HasNoKey();
+            modelBuilder.Entity<RevisionEditByIdCommand>().HasNoKey();
+            modelBuilder.Entity<CarCreateCommand>().HasNoKey();
+            modelBuilder.Entity<CarEditByIdCommand>().HasNoKey();
```

Comment:
You only need to use this for classes that read from SQL query outputs (like results). Commands never reach EntityFramework (only their properties do but that's fine).
```suggestion
```